src/Avalonia.Controls/Window.cs

```diff
@@ -183,6 +183,15 @@ public WindowIcon Icon
             set { SetValue(IconProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the startup location of the window.
+        /// </summary>
+        public WindowStartupLocation WindowStartupLocation
```

Comment:
Is there a reason you didn't change this? I notice you committed fixes for the other comments.